src/scala/ly/stealth/mesos/kafka/Executor.scala

```diff
@@ -92,6 +95,25 @@ object Executor extends org.apache.mesos.Executor {
       }
     }
 
+    def startCollectingMetrics(): Unit = {
+      new Thread {
+        setName("BrokerMetrics")
+        def send(driver: ExecutorDriver, metrics: Broker.Metrics): Unit = {
+          driver.sendFrameworkMessage(JSONObject(Map("metrics" -> metrics.toJson)).toString().getBytes)
+        }
+        override def run(): Unit = {
+          while (true) {
+            try {
+              BrokerServer.Metrics.collect.foreach(metrics => send(driver, metrics))
```

Comment:
we need a way to message them without restarting (through the framework api) so we can change (add/remove) metrics getting polled